components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -480,13 +481,6 @@ - (UITabBarItem *)tabBarItemForPoint:(CGPoint)point {
 
 #pragma mark - Touch handlers
 
-- (void)didTouchDownButton:(UIButton *)button {
-  MDCBottomNavigationItemView *itemView = (MDCBottomNavigationItemView *)button.superview;
-  CGPoint centerPoint =
-      CGPointMake(CGRectGetMidX(itemView.inkView.bounds), CGRectGetMidY(itemView.inkView.bounds));
-  [itemView.inkView startTouchBeganAnimationAtPoint:centerPoint completion:nil];
-}
-
```

Comment:
Can we move these deletions to a separate PR?

Reply:
yes! will do